Python/Tests/DjangoUITests/DjangoUITests.csproj

```diff
@@ -85,11 +85,6 @@
     <Reference Include="Microsoft.VisualStudio.Threading">
       <SpecificVersion>False</SpecificVersion>
     </Reference>
-    <Reference Include="Microsoft.Web.Core" Condition=" '$(VSTarget)' == '15.0' " />
```

Comment:
The references for 15.0 have been removed, but the build pipelines still use 15.0 VS. Can you confirm if everything works when building with 15.0?

Same for the other changes.

Reply:
It still builds successfully on my pipeline which uses pool VSEng-MicroBuildVS2017, see the latest build log from PTVS-GitHub-Huguesv-Dev16 pipeline.

We do not need to target VS 2017 anymore from master branch, so I'm not maintaining that anymore (on master or 16.x branches).